jobs/redis/templates/config/redis.conf.erb

```diff
@@ -105,9 +105,19 @@ databases 16
 #
 #   save ""
 
-save 900 1
-save 300 10
-save 60 10000
+<% if_p('redis_save_intervals') do
+    redis_save_intervals = p('redis_save_intervals')
+
+    if redis_save_intervals.length > 0
+        redis_save_intervals.each do | command |
```

Comment:
shouldn't be an issue I think, as the same syntax is already used here: https://github.com/cloudfoundry-community/redis-boshrelease/blob/master/jobs/redis/templates/config/redis.conf.erb#L172-L174